Engines/Wine/Verbs/Windows XP SP 3/script.js

```diff
@@ -10,17 +10,11 @@ const {remove, fileName} = include("utils.functions.filesystem.files");
  * @returns {Wine} Wine object
  */
 Wine.prototype.sp3extract = function (fileToExtract) {
-    var that = this;
-    that._targetDirectory = this.system32directory();
+    const targetDirectory = this.system32directory();
 
-    this.targetDirectory = function (targetDirectory) {
-        that._targetDirectory = targetDirectory;
-        return that;
-    };
-
-    var setupFile = new Resource()
+    const setupFile = new Resource()
         .wizard(this.wizard())
-        .url("http://freeware.epsc.wustl.edu/Win/XP_SP3/WindowsXP-KB936929-SP3-x86-ENU.exe")// Just a test, the URL needs to be fixed
```

Comment:
I hope the comment was just a copy&paste error or something that way accidentally forgotten

Reply:
I don't remember ... sorry